chemicaltoolbox/sucos/sucos.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python
+"""
+Basic SuCOS scoring. Allows a set of molecules from a SD file to be overlayed to a reference molecule,
+with the resulting scores being written as properties in the output SD file.
+
+SuCOS is the work of Susan Leung.
+GitHub: https://github.com/susanhleung/SuCOS
```

Comment:
Do you want to add that this is MIT licensed?
We could also create a conda package out of that ....

Reply:
Mentioned MIT license: https://github.com/tdudgeon/galaxytools/commit/10f9f7c36998af3b4104e02ddda1a1bb1ccb13eb